tests/test_trading_calendar.py

```diff
@@ -570,10 +570,12 @@ def test_sessions_in_range(self):
                                             second_session_label)
         )
 
-    def _get_session_block(self):
-        # Try to find and return a (full session, early close session, full
-        # session) block.
-
+    def get_session_block(self):
```

Comment:
It might be helpful to have this raise an error telling the caller to override it if the random sessions aren't contiguous. Otherwise a new calendar implementor will have to trace back a confusing failure to this.

Reply:
This function will always return 3 sessions in a row; I suppose they may not be contiguous in the sense that a weekend could be in between, but as far as I can tell that shouldn't actually matter. So maybe I should just change the wording of the doc string to say "3 in a row" or "continuous".